Backpack/src/main/java/net/skyscanner/backpack/chip/internal/Utils.kt

```diff
@@ -40,16 +40,19 @@ internal fun chipRoundedRect(
 internal fun chipColors(
   selected: Int,
   default: Int,
+  pressed: Int,
   disabled: Int,
 ) = ColorStateList(
   arrayOf(
     intArrayOf(android.R.attr.state_enabled, android.R.attr.state_selected),
-    intArrayOf(android.R.attr.state_enabled),
     intArrayOf(-android.R.attr.state_enabled),
+    intArrayOf(android.R.attr.state_pressed),
```

Comment:
again the order is intentional here - pressed only applies to the non-selected & enabled state